fgpyo/fasta/reference_set_builder.py

```diff
@@ -0,0 +1,174 @@
+"""
+Classes for generating Fasta files and records for testing
+----------------------------------------------------------------
+"""
```

Comment:
Can you add examples for the user similar to https://github.com/fulcrumgenomics/fgpyo/blob/main/fgpyo/sam/__init__.py

Reply:
Do you want the examples in ref_set_builder.py or __init__.py?

Reply:
Typically in the module where the class is defined, so in `ref_set_builder.py`

Reply:
Added examples in commit cb8ac832a05ba4a3ad304edb2de19c57d6f9b8f3